### PR TITLE
Notes changes

### PR DESCRIPTION
**Pull request by NateBigStone**
Display rating stars for each user. #39

### CODE REVIEW

**Comment by sunnycake** [comment]
@NateBigStone Not sure what output you're receiving, but mine is showing all reviews with 5 stars and throwing operational error when I click on the user (http://127.0.0.1:8000/user/profile) name to view their individual notes.

**Comment by NateBigStone** [comment]
@sunnycake The stars are different colors. You may need to clear your cache for the css to update.
Not good practice, but I could put the css in-line ?
Currently investigating the user/profile

**Comment by NateBigStone** [comment]
@sunnycake  user/profile works for me. Maybe you need to do a makemigration to get the profile table added to the db ?

**Comment by sunnycake** [comment]
@NateBigStone if all is working good on your end then all is good. Probably because I forgot to make migrations. I can't test it right now (at hospital) but if it's not merged yet when I'm back then I'll look at it again. Thank you!

File: lmn/templates/lmn/notes/_note.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<div id="note_{{ note.pk }}" onclick="location.href='/notes/detail/{{ note.pk }}';">
+    <h3 class="note-title">{{ note.title }}</h3>
```
Comment (yp0975eu) [inline lmn/templates/lmn/notes/_note.html]: should this link be moved to note title?

**Review by yp0975eu — APPROVED** [review]

File: lmn/templates/lmn/notes/_note.html (hunk 1 above)
Comment (NateBigStone) [reply lmn/templates/lmn/notes/_note.html]: Changed

**Pull request merged**